pkg/operator/ceph/operator.go

```diff
@@ -162,6 +163,11 @@ func (o *Operator) Run() error {
 			close(stopChan)
 			o.clusterController.StopWatch()
 			return nil
+		case err := <-mgrErrorChan:
+			logger.Errorf("gave up to run the operator. %v", err)
+			close(stopChan)
```

Comment:
suggestion: Factor out lines 163-164 to a helper method to share the close functionality with lines 168-169.